testing/sdk_tasks.py

```diff
@@ -248,11 +248,12 @@ def get_summary(with_completed: bool = False, task_name: Optional[str] = None) -
     )
     if task_name:
         output = list(filter(lambda t: t.name == task_name, output))
-    log.info(
-        "Task summary (with_completed={}, task_name=[{}]):\n- {}".format(
-            with_completed, task_name, "\n- ".join([str(e) for e in output])
-        )
-    )
+    # Commenting below two lines to avoid huge output
```

Comment:
Change the log to debug instead of commenting it out

Reply:
This is useless huge log. We earlier also commented out in the dcos-commons repo. Finding real problem does not help with this log. It just floods the output and increase log file size.

Reply:
Debug logs aren't printed by default, right? So you can change it to debug, it won't be printed then

Reply:
I did not see any log.debug method, please point me to the method you wanted to use here.

Reply:
It is part of the logger. You can just use `log.debug()`

https://docs.python.org/2/library/logging.html#logging.debug